package.json

```diff
@@ -22,6 +22,7 @@
     "build": "npm run clean && npm run renderTemplates && tsc && eslint dist --ext .ts,.js --quiet --fix",
     "clean": "rm -rf renderedTemplates dist",
     "depcheck": "depcheck",
+    "diffApis": "raml-toolkit diff --dir updateApisTmp/oldApis apis -o updateApisTmp/ramlDiff.json",
```

Comment:
Do we have any test cases to cover this scenario?

Reply:
The diff functionality has tests  written in raml-toolkit